### PR TITLE
Clarify readme for tiny example

### PR DESCRIPTION
**Pull request by linsun**
I ran into a few issues when trying the tiny example with kubernetes module only today.
1) The doc was not clear as to tiny installer vm vs creating your own installer env.
2) i assumed it was using root user to run Ansible scripts, which caused not able to SSH into the VMs issue.

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
@linsun this change undoes some other recent work.  Can you start over from the current master?

File: examples/tiny-example.md [hunk 1]
```diff
@@ -97,26 +97,19 @@ most of the contents of the OpenRadiant repository in `~/openradiant`.
...
 
-See
+If you have created the installer VM using [Create Installer VM](#create-installer-vm), you can skip this section.   To manually create the installer, see
 [the general documentation of the installer machine](../README.md#the-installer-machine)
 for the general story.  Following is one concrete realization of that
 story for this example.
 
-If you are running Ubuntu on your installer, you may need to install
-the following python packages:
+If you are running Ubuntu in your host, you may need to install the following
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: This change is undoing a more recent change.

File: examples/tiny-example.md [hunk 2]
```diff
@@ -97,26 +97,19 @@ most of the contents of the OpenRadiant repository in `~/openradiant`.
...
 
-OTOH, if you are running MacOS 10 on your installer then you probably
-need to:
-
-```bash
-brew install gnu-tar
-```
-
-In any case, install ansible and its `netaddr` module:
+Install ansible and its `netaddr` module:
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: This change is undoing a recent change, please don't.

File: examples/tiny-example.md [hunk 3]
```diff
@@ -136,16 +129,24 @@ pip install --upgrade ansible
 
 Use Ansible on the installer machine to begin the process of deploying
 an environment.  This will create the certificates and keys that are
-common throughout the environment, and deploy the API proxy.
+common throughout the environment.  Someday soon this will also deploy
+the API proxy.
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: This undoes a recent change.  Please don't.

File: examples/tiny-example.md [hunk 4]
```diff
@@ -136,16 +129,24 @@ pip install --upgrade ansible
 
 Use Ansible on the installer machine to begin the process of deploying
 an environment.  This will create the certificates and keys that are
-common throughout the environment, and deploy the API proxy.
+common throughout the environment.  Someday soon this will also deploy
+the API proxy.   
+
+Use vagrant ssh to ssh into the installer machine:
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: This is correct only if the installer VM was used.

File: examples/tiny-example.md [hunk 5]
```diff
@@ -136,16 +129,24 @@ pip install --upgrade ansible
...
+vagrant ssh installer-tiny
+```
+
+Execute the following ansible scripts as the vagrant user:
 
 ```bash
 ( cd ansible; \
   ansible-playbook -v -i ../examples/envs/dev-vbox/radiant01.hosts env-basics.yml \
-      -e "envs=../examples/envs env_name=dev-vbox" )
+      -e "envs=../examples/envs env_name=dev-vbox cluster_name=dev-vbox-radiant01" )
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: The `env-basics.yml` playbook does not need the `cluster_name` variable to be set.

File: examples/tiny-example.md [hunk 6]
```diff
@@ -136,16 +129,24 @@ pip install --upgrade ansible
...
 ```
 
 Use Ansible on the installer machine to deploy an OpenRadiant shard on
-the target machines.
+the target machines, using the vagrant user.
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: The change adds a dangling modifier.  Did you intend "using the vagrant user" to modify "Use Ansible" or "deploy"?  Please reword to be clear, and not assume that the installer VM was used.

**Comment by mrsabath** [comment]
In fact, I suggest putting these changes on hold until @akanso finishes the updates to the examples. We are redesigning the tiny example.

**Comment by MikeSpreitzer** [comment]
Yes, all of the changes here either intersect with the example revisions we just agreed or are mistaken reverts.

**Comment by linsun** [comment]
Mike, thanks for the review.  I had accidentally reverted a few changes.  I also addressed your feedback on vagrant user only when installer VM used.

File: examples/tiny-example.md [hunk 7]
```diff
@@ -138,10 +138,17 @@ Use Ansible on the installer machine to begin the process of deploying
 an environment.  This will create the certificates and keys that are
 common throughout the environment, and deploy the API proxy.
 
+If installer VM is used, use vagrant ssh to ssh into the installer machine:
+```bash
+vagrant ssh installer-tiny
+```
+
+If installer VM is used, execute the following ansible scripts as the vagrant user:
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: The playbook needs to be executed, regardless of whether the installer VM is being used.

**Comment by MikeSpreitzer** [comment]
You missed my comments on new line 105 and new line 151, and I added a comment at new line 146.

**Comment by linsun** [comment]
Thank you Mike, I *hopefully* have reworked your comment on 101, 151, and the new comment on running scripts for non-vagrant vm installer.  For the comment in 151, I hadn't tried it.  In the previously incorrect git clone cmd, I had to add the cluster_name to get it working.

File: examples/tiny-example.md [hunk 8]
```diff
@@ -138,14 +138,17 @@ Use Ansible on the installer machine to begin the process of deploying
...
+vagrant ssh installer-tiny
+```
+
 ```bash
 ( cd ansible; \
   ansible-playbook -v -i ../examples/envs/dev-vbox/radiant01.hosts env-basics.yml \
       -e "envs=../examples/envs env_name=dev-vbox" )
 ```
 
-Use Ansible on the installer machine to deploy an OpenRadiant shard on
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: You are undoing the recent addition of this remark.  Please don't.

File: examples/tiny-example.md [hunk 9]
```diff
@@ -138,14 +138,17 @@ Use Ansible on the installer machine to begin the process of deploying
 an environment.  This will create the certificates and keys that are
 common throughout the environment, and deploy the API proxy.
 
+If you use the installer VM created from [Create Installer VM](../tiny-example.md#create-installer-vm) as the installer machine, use vagrant ssh to ssh into the installer VM and execute the Ansible scripts as the *vagrant* user:
```
Comment (MikeSpreitzer) [inline examples/tiny-example.md]: What if "you" are _not_?
Why repeat https://github.com/containercafe/containercafe/blob/master/examples/tiny-example.md#fyi-on-ssh-to-vagrantvirtualbox-vms here?

**Comment by MikeSpreitzer** [comment]
Please re-read the latest version of this doc.  You are still undoing recent work.

**Comment by linsun** [comment]
Thank you Mike!  Hopefully I didn't revert anything this time!   I found a way to preview links to see if it is working so I can avoid the commits to try the links in the future. :)  Overall, I wish we could have less doc for standing up radiant in local env in the simplest scenario... I think we offer lots of choices which is great when people need choices/options, but it does took quite long to stand up a simple env.   It took me 1-1.5 days to stand up the working env for Kube.

**Comment by MikeSpreitzer** [comment]
Yes, Lin, some of us discussed the example yesterday and decided to re-org into one example that is focused on showing what it is like for Maureen to use an OpenRadiant environment and another example that shows an operator (like the Conductors squad) what it is like to operate an OpenRadiant environment.  @akanso is working on the first of those two.

**Comment by linsun** [comment]
Thank you Mike for merging / reviewing!   It would be great if we could make this simpler for people to try open radiant quickly, maybe even have a simple developer version.

**Pull request merged**